NFC_EEPROM/source/main.cpp

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+#include "mbed.h"
```

Comment:
No, avoid to include mbed.h directly as it pulls out a lot of includes and pollute the global namespace with its `using namespace std` an `using namespace mbed statements`. 

If you need something specific. Include it and import it in the local namespace (if your in a cpp file) if you want shorter names. 

```c++ 
#include "Drivers/DigitalIOut.h"

using mbed::DigitalOut;
```